config/config.go

```diff
@@ -1259,6 +1269,47 @@ func migrateConfigSpecialFeature1(v *viper.Viper) {
 	}
 }
 
+func migrateConfigTCF2PurposeFlags(v *viper.Viper) {
+	migrateConfigTCF2EnforcePurposeFlags(v)
+	migrateConfigTCF2PurposeEnabledFlags(v)
+}
+
+func migrateConfigTCF2EnforcePurposeFlags(v *viper.Viper) {
+	for i := 1; i <= 10; i++ {
```

Comment:
How do we know this for loop needs to run 10 times specifically?

Reply:
There are ten purposes, each of which can be configured in the host config and account config. Each purpose config can be specified as `gdpr.tcf2.purposeX` where X is 1-10.